cpp/oneapi/dal/algo/rbf_kernel/backend/gpu/compute_kernel_dense_dpc.cpp

```diff
@@ -29,46 +27,113 @@ using input_t = compute_input<task::compute>;
 using result_t = compute_result<task::compute>;
 using descriptor_t = detail::descriptor_base<task::compute>;
 
-namespace daal_rbf_kernel = daal::algorithms::kernel_function::rbf;
-namespace interop = dal::backend::interop;
+namespace pr = dal::backend::primitives;
 
 template <typename Float>
-using daal_rbf_kernel_t =
-    daal_rbf_kernel::internal::KernelImplRBFOneAPI<daal_rbf_kernel::defaultDense, Float>;
+inline auto compute_rbf_values(sycl::queue& queue,
+                               const pr::ndview<Float, 1>& sqr_x_nd,
+                               const pr::ndview<Float, 1>& sqr_y_nd,
+                               pr::ndview<Float, 2>& res_nd,
+                               double sigma,
+                               const dal::backend::event_vector& deps = {}) {
+    const std::int64_t x_row_count = sqr_x_nd.get_dimension(0);
+    const std::int64_t y_row_count = sqr_y_nd.get_dimension(0);
```

Comment:
Please, put an `ONEDAL_ASSERT` that `res_nd` shall contain at least `x_row_count*y_row_count` elements